src/proto/cardset.proto

```
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package cardset;
```

Buf (1.66.0)

[error] Files with package "cardset" must be within a directory "cardset" relative to root but were in directory "src/proto".

(PACKAGE_DIRECTORY_MATCH)